### DISCUSSION TITLE
!Clogs / !Total collection logs

### DISCUSSION THREAD

**Discussion by itswatkins** [runelite/runelite#17881] [Ideas]
Would be neat if there was a !clogs command to just list out {user} has a total of x/1518 collection logs.

Simple but would be neat, and have seen a few ppl in clans bring it up also :)

**Comment by Felanbird** [comment]
This is already a feature of the hub plugin [collection log](https://runelite.net/plugin-hub/show/collection-log)

**Reply by itswatkins** [reply]
> This is already a feature of the hub plugin [collection log](https://runelite.net/plugin-hub/show/collection-log)

Ah okay I've literally never seen that feature work, not sure if it's bugged or just refuses to work for me despite all other !log commands working

**Reply by IromTypo** [reply]
> > This is already a feature of the hub plugin [collection log](https://runelite.net/plugin-hub/show/collection-log)
> 
> Ah okay I've literally never seen that feature work, not sure if it's bugged or just refuses to work for me despite all other !log commands working

https://collectionlog.net/faq
"Why is my collection log not updating"

It's takes a couple minutes to set up
